source/UIAHandler/_remoteOps/instructions/general.py

```diff
@@ -0,0 +1,65 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2023-2024 NV Access Limited
+
+"""
+This module contains the instructions that operate on all object types.
+Including to set a value, compare two values, and stringify a value.
+"""
+
+
+from __future__ import annotations
+from dataclasses import dataclass
+from .. import lowLevel
+from .. import builder
+from ._base import _TypedInstruction
+
+
+@dataclass
+class Set(_TypedInstruction):
```

Comment:
perhaps this should be renamed Assign as it this could be confused with a `set(), {1, 2, 3}`

Reply:
Again, this reflects Microsoft's low-level instruction name though I see your point. However, these instructions are only to be used internally by the _remoteOps package, and not by code that uses remote operations through out NVDA.